website/content/en/docs/overview/project-layout.md

```diff
@@ -0,0 +1,91 @@
+| `bundle/tests/` | This directory has the [Scorecard][scorecard] tests shipped with your operator bundle. |
+| `config/` | Contains configuration files to launch your project on a cluster. See that plugins might use it to provide their facilities. For example, for the CLI be able to help you out create your operator bundle it will look for the CRD's and CR's which are scaffold in this directory. Also, see that you will find here as well all [Kustomize][Kustomize] YAML definitions. |
+| `config/crd/` | Contains the [Custom Resources Definitions][k8s-crd-doc] |
+| `config/default/` | contains a [Kustomize base][kustomize-base] for launching the controller in a standard configuration. |
+| `config/manager/` | Contains the manifests to launch your operator project as pods on the cluster |
+| `config/manifests/` | Contains the base to generate your OLM manifests in the bundle directory |
+| `config/prometheus/` | Contains the manifests required to enable project to serve metrics to [Prometheus][kb-metrics] |
+| `config/scorecard/` | Contains the manifests required to allow you test your project with [Scorecard][scorecard] |
+| `config/rbac/` | Contains the [RBAC][k8s-rbac] permissions required to run your project |
+| `config/samples/` | Contains the [Custom Resources][k8s-cr-doc] |
+| `hack/` | Contains the file used to scaffold the lincense in the head of your project files ||
+| `bundle.Dockerfile` | The project [bundle][bundle] Dockerfile |
+
+### Ansible
+
+Now, let's check the files and directories which are specific for Ansible.
+
+| File/Directory | Description | 
+| ------ | ----- |
+|`config/testing/` | Manifest files that can help you to test your project. For example, to change the configuration of the image policy for your [Molecule tests][ansible-test-guide] tests and to enable the Ansible logs in the debug level. |
+|`molecule/` | Contain the manifests for your [Molecule][molecule] tests |
+|`molecule/default` | Contains the default [Molecule][molecule] task |
+|`molecule/kind` | Contains the [Molecule][molecule] task to be executed on the cluster |
+|`playbooks/` | Contains the Ansible playbooks|
+|`roles/` | Contains the Ansible role files for each Kind scaffold |
+|`requirements.yml` | This file specifies Ansible dependencies that need to be installed for your operator to function. |
+|`watches.yaml` | Use to configure the [Ansible watches][ansible-watches]. |
+
+### Golang 
+
+Now, let's check the files and directories which are specific for Helm.
+
+| File/Directory | Description |
+| ------ | ----- |
+|`api/` | Contains the api definition |
+|`config/certmanager` |  Contains the Kustomize manifests which configure the cert-manager which are used with the Webhooks |
+|`config/webhook` | Contains the Kustomize manifests to configure the Webhook |
+|`controllers` |  Contains the controllers |
+|`main.go` | Its implement the project initialization |
+
+### Helm 
```

Comment:
same thing for helm. can we remove this to point it here https://sdk.operatorframework.io/docs/building-operators/helm/reference/project_layout/ to avoid duplication?

Reply:
great point varchar. Let's remove the helm one and keep it centralized.